examples/check-if-opened-with-testcafe/README.MD

```diff
@@ -0,0 +1,17 @@
+# Check Whether Web Page Is Opened With TestCafe
+
+**Test Code**: [index.js](index.js)
+**Tested Page**: [index.html](index.html)
+
+You may need to make adjustments to your application depending on whether it is tested with TestCafe. This example demonstrates how to let your application know whether it is opened with TestCafe.
```

Comment:
"You may need to make adjustments to your application depending on whether it is tested with TestCafe" This is an 'empty' sentence, and by that I mean it lacks real meaning. For example, "depending on whether it is tested with TestCafe": this topic is about TestCafe, so why would it depend on this? "You may need to make adjustments": which 'adjustments'? Change options? To be honest, it is difficult to tell what you are trying to say here. Do you need to change your application before you can test it in TestCafe? Why do you need to 'let your application know' that you opened it in TestCafe?